integreat_cms/static/src/js/regions/fake_disabled_checkboxes.ts

```diff
@@ -1,7 +1,28 @@
+const setZammadUrlDisabledState = (zammadWrapper: HTMLElement, zammadUrl: HTMLInputElement) => {
+    const checkboxes = zammadWrapper.querySelectorAll('input[type="checkbox"].fake-disable-region');
+    if (checkboxes.length) {
+        zammadUrl.classList.add("pointer-events-none");
+    } else {
+        zammadUrl.classList.remove("pointer-events-none");
+    }
+};
+
 window.addEventListener("load", () => {
     const checkboxes = <HTMLInputElement[]>(<any>document.querySelectorAll(".fake-disable-region"));
     checkboxes.forEach((checkbox) => {
         const label = document.querySelector(`label[for='${checkbox.id}']`);
         label.classList.add("pointer-events-none");
     });
+
+    const zammadWrapper = document.querySelector("#zammad-offers-wrapper") as HTMLElement;
+    const zammadUrl = document.querySelector("#id_zammad_url") as HTMLInputElement;
+    if (!zammadWrapper || !zammadUrl) {
+        return;
+    }
+
+    const zammadCheckboxes = <HTMLInputElement[]>(<any>zammadWrapper.querySelectorAll('input[type="checkbox"]'));
+    zammadCheckboxes.forEach((checkbox) => {
+        checkbox.addEventListener("change", () => setZammadUrlDisabledState(zammadWrapper, zammadUrl));
```

Comment:
> so this means: "when checking a zammad offer the url field should be disabled"? This does not work for me. Am I missing sth.?

Not sure I understand the question correctly - the URL input field will only be disabled after you check a Zammad form checkbox, then save (which is only possible if a Zammad URL is set).

That's just to prevent removal of the URL while any Zammad offers are still enabled.

Reply:
> I'm missing the checkbox?

Have you deleted the Zammad Offer? 😅